README.md

```diff
@@ -77,4 +77,20 @@ You can filter which parts of the code your want to instrument using the option
 The exclusion file should follow the following format:
 
 - one exclusion rule per line
-- each line should be of the form `fctname pathsymbol` , where `fctname` is the function name, and `pathsymbol` is the path for the symbol: the shared library or the executable. 
+- each line should be of the form `pathsymbol fctname` , where `fctname` is the function name, and `pathsymbol` is the path for the symbol: the shared library or the executable. 
```

Comment:
inverse the rest of the sentence also please :  each line should be of the form `pathsymbol fctname` , where pathsymbol is ... and fctname is ....